src/codeu/chat/common/User.java

```diff
@@ -88,4 +95,29 @@ private static int hash(User user) {
     hash+=user.creation.hashCode();
     return hash;
   }
+
+  public void addUserInterest(Uuid user)
```

Comment:
Consider renaming Uuid paramters to include "Id" at the end (userId, convoId, etc)